llvm/test/CodeGen/AMDGPU/has_cache_straddle.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
```

Comment:
This is unusual and we probably shouldn't need python in these tests. If it's going to exist it needs to be in an Inputs subdirectory